base_site/index.html

```diff
@@ -196,15 +221,16 @@ <h1 class="heading-style-h1 text-color-gray">Technical Documentation</h1>
                 <div class="footer_link-list-header">Solutions</div>
                 <a href="https://docs.quantinuum.com/nexus/index.html" class="footer_link">Quantinuum Nexus</a>
                 <a href="https://docs.quantinuum.com/inquanto/index.html" class="footer_link">InQuanto</a>
-                <a href="https://docs.quantinuum.com/tket/index.html" class="footer_link">TKET</a>
+                <a href="https://tket.quantinuum.com/" class="footer_link">TKET</a>
+                <a href="https://docs.quantinuum.com/lambeq" class="footer_link">Lambeq</a>
               </div>
-              <div id="w-node-_2d500eba-afac-a112-5c0c-095934006ea3-3bd06415" class="footer_link-list">
+              <div id="w-node-faa91be7-6526-4ea7-e789-d58353b2f232-c2a09c1d" class="footer_link-list">
                 <div class="footer_link-list-header">Hardware</div>
-                <a href="https://www.quantinuum.com/hardware/h1" class="footer_link">System Model H1</a>
-                <a href="https://www.quantinuum.com/hardware/h2" class="footer_link">System Model H2</a>
+                <a href="https://docs.quantinuum.com/h-series/index.html" class="footer_link">System Model H1</a>
+                <a href="https://docs.quantinuum.com/h-series/index.html" class="footer_link">System Model H2</a>
```

Comment:
```suggestion
                <a href="https://www.quantinuum.com/hardware/h1" class="footer_link">System Model H1</a>
                <a href="https://www.quantinuum.com/hardware/h2" class="footer_link">System Model H2</a>
```

These look like they should stay the same, as they're links to specific product pages - another one to feed back to Luke?